python/ray/data/_internal/block_batching.py

```diff
@@ -148,7 +148,8 @@ def _format_batch(batch: Block, batch_format: str) -> BatchType:
     else:
         raise ValueError(
             f"The given batch format: {batch_format} "
-            f"is invalid. Supported batch type: {BatchType}"
+            f"is invalid. Supported batch type: "
+            f"{'default', 'pandas', 'pyarrow', 'numpy'}"
```

Comment:
shall we be more specific for the exact batch type, like

```
Supported batch type: {'default', 'pandas.DataFrame', 'pyarrow.Table', 'numpy.ndarray'}
```

Reply:
I don't think so, because those aren't valid parameter values.

For example, if you call `iter_batches(batch_format="arrow")`, you'd get the error message

> The given batch format: arrow is invalid. Supported batch type: {'default', 'pandas.DataFrame', 'pyarrow.Table', 'numpy.ndarray'}

This error isn't useful because it doesn't tell the user how to fix their code. You can't do `iter_batches(batch_format="pyarrow.Table")`.

Reply:
We could clarify by stating 

> Supported `batch_format` values:

instead of

> Support batch type:

Reply:
agreed, let's change to `Supported batch_format values: `.